torchvision/models/resnet.py

```diff
@@ -204,6 +204,8 @@ def __init__(
 
     def _make_layer(self, block: Type[Union[BasicBlock, Bottleneck]], planes: int, blocks: int,
                     stride: int = 1, dilate: bool = False) -> nn.Sequential:
+        if blocks <= 0:
+            return nn.Identity()
```

Comment:
This needs to be sequential for typing reasons:
```suggestion
            return nn.Sequential(nn.Identity())
```